sciencebeam/pipelines/__init__.py

```diff
@@ -6,6 +6,20 @@
 
 from sciencebeam.utils.config import parse_list
 
+class FieldNames(object):
```

Comment:
Was there a reason for opting for a `class` rather than just a `dict` for these? on the surface, they just look like key-value pairs.

Reply:
I thought that's a common pattern to define scoped constants / enum values (using proper enums would also be an option).